src/platform/Zephyr/ConfigurationManagerImpl.cpp

```diff
@@ -206,7 +206,7 @@ void ConfigurationManagerImpl::DoFactoryReset(intptr_t arg)
 
 CHIP_ERROR ConfigurationManagerImpl::GetPrimaryWiFiMACAddress(uint8_t * buf)
 {
-#if CHIP_DEVICE_CONFIG_ENABLE_WIFI
+#if defined(CHIP_DEVICE_CONFIG_ENABLE_WIFI)
```

Comment:
All those, `CHIP_DEVICE_CONFIG_XXX` could be set to 0 as well, so checking they are defined only is error-prone. Alternatively, we should just make sure in `CHIPDevicePlatformConfig.h` that these are always defined and set to either 1 or 0.

Reply:
fixed